FSEParser_V3.1.py

```diff
@@ -28,14 +28,12 @@
 import re
 import datetime
 import sqlite3
-import time
 import json
-import csv, codecs, cStringIO
+import ujson

```

Comment:
Fails on import if the module does not exist. A try, except would be useful here so that ujson or json can be used.

Reply:
awesome thanks I will put that in!